Bottom: cb059b49c84ceddc5e238021acdda23353d2fa9f
Top:    9362164b4afbd0e4252e5b9cb847fc1c53fcab60
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-05-15 18:09:26 +0100

find.c: The User varaible needs to die...

Reduce its usage to where absolutely necessary... oh, what a surprise,
another variable passing state down into add_object_to_list ()


---

diff --git a/src/find.c b/src/find.c
index 3747233..1c755f7 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3090,7 +3090,7 @@ LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int flag,
    */
   ListStart (type, ptr1, ptr2, ptr3, flag);
   DoIt (flag, AndRats, AndDraw);
-  if (User)
+  if (AndDraw)
     IncrementUndoSerialNumber ();
   User = false;
